addons/l10n_sa_edi/tests/test_edi_zatca.py

```diff
@@ -215,3 +215,44 @@ def test_invoice_standard_with_future_time(self):
         errors = self.edi_format.with_user(self.user_saudi.id)._check_move_configuration(move)
         msg = '- Please, make sure the invoice date is set to either the same as or before Today.'
         self.assertTrue(msg in errors)
+        
+    def test_invoice_type_code(self):
+        move_us_company = self._create_invoice(
+            name='INV/2024/00014',
+            date='2024-02-20',
+            date_due='2024-02-28',
+            partner_id=self.partner_us,
+            product_id=self.product_a,
+            price=320.0,
+            user=self.user_saudi,
+        )
+        
+        invoice_type_code = self.env['account.edi.xml.ubl_21.zatca']._l10n_sa_get_invoice_transaction_code(move_us_company)
+        # Not a simplified invoice since the partner is not from saudi arabia and a company
```

Comment:
Actually in your test, it's from Saudi Arabia, either adapt the comment, either change for `partner_us`.

Reply:
I can't read, forget that